server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -22,6 +22,7 @@ grant codeBase "${codebase.elasticsearch-secure-sm}" {
 grant codeBase "${codebase.elasticsearch}" {
   // needed for loading plugins which may expect the context class loader to be set
   permission java.lang.RuntimePermission "setContextClassLoader";
+  permission java.net.SocketPermission "*", "resolve, listen, accept";
```

Comment:
Side note: This is adding back SocketPermission where we previously removed it when moving network implementations to modules. This is better than it was before (previously a bare grant, so all code in ES could bind sockets), but this is still pretty broad (all of server). I just want to caution that we should see (in a followup maybe, or in future refactorings, say with modularization) how we can further isolate the readiness service so that it can be the only thing that needs this additional permission.

We should however limit to localhost still? Also, note that `resolve` is not needed, it is still in the bare grant.